test_src/sljitTest.c

```diff
@@ -7387,8 +7600,6 @@ static void test72(void)
 		return;
 	}
 
-	/* Next test. */
```

Comment:
Why are these removed?

Reply:
At some places they seemed unnecessary. Should I undo these?

Reply:
No, you are right. I haven't checked the full context.